packages/ui/cypress/tests/multisig-creation.cy.ts

```diff
@@ -93,6 +127,14 @@ describe('Multisig creation', () => {
       acceptMultisigCreation()
 
       landingPage.infoMultisigCreated().should('be.visible')
+      accountDisplay.nameLabel().should('contain.text', multisigName)
+      multisigPage
+        .transactionList()
+        .should('be.visible')
+        .within(() => {
+          multisigPage.pendingTransactionItem().should('have.length', 2)
```

Comment:
Why? This is not expect at all.

Reply:
@Tbaut That's what I got when creating a pure. It's probably a bug as we returned the pending transaction which should be associated with previous test case "create Pure". Did I miss something? 

<img width="1221" alt="Screenshot 2023-12-15 at 18 06 14" src="https://github.com/ChainSafe/Multix/assets/8141917/6e29e624-4701-4aee-8644-84d1c0b879d2">

Reply:
if you create twice the same multisig, with the same account, once with a pure, once without, then this is expected, but this test is far from what a normal user would do. We should create 2 different multisigs, one with a pure, another one without, the 2 multisigs should be different.